### PR TITLE
Feat: add dropdown component

### PR DESCRIPTION
**Pull request by RaoufDannaoui1**
we should add array of item to display them in the drop menu

### CODE REVIEW

**Review by osamaakb — COMMENTED** [review]
What will you do when the menu item is clicked?

**Comment by RaoufDannaoui1** [comment]
On click it will filter the activities for example and show only the ones that are in  this city

**Comment by osamaakb** [comment]
> On click it will filter the activities for example and show only the ones that are in this city

Yes but I don't see the onClick function implemented, can you implement it please

File: src/Components/Dropdown/Dropdown.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { Menu, Dropdown, Button } from 'antd';
+import { DownOutlined } from '@ant-design/icons';
+import './Dropdown.css';
+export default function DropdownButton({
+  elementArray,
+  activitiesArray,
```
Comment (osamaakb) [inline src/Components/Dropdown/Dropdown.js]: why are you sending the activities here?

File: src/Components/Dropdown/Dropdown.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  elem,
+  dropdownTitle,
+}) {
+  const [elementChecked, setElementChecked] = useState(dropdownTitle);
+  const handleClick = (element) => {
+    console.log(element.item.props.value);
+    let value = element.item.props.value;
+    setElementChecked(value);
+    activitiesArray.forEach((act) => {
+      if (act[elem] == value) {
```
Comment (osamaakb) [inline src/Components/Dropdown/Dropdown.js]: this logic should be done in the page that's using the dropdown not here

File: src/Components/Dropdown/Dropdown.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { Menu, Dropdown, Button } from 'antd';
+import { DownOutlined } from '@ant-design/icons';
+import './Dropdown.css';
+export default function DropdownButton({
+  elementArray,
+  activitiesArray,
+  handleSelect,
+  elem,
```
Comment (osamaakb) [inline src/Components/Dropdown/Dropdown.js]: we don't need this one too

**Review by osamaakb — CHANGES_REQUESTED** [review]

**Review by osamaakb — APPROVED** [review]

**Pull request merged**